docker/archimedes-base/Dockerfile

```diff
@@ -7,7 +7,7 @@ WORKDIR /app
 ENV DOCKER_VERSION 19.03.12
 ENV PYTHON_VERSION 3.8
 ENV DOCKERIZE_VERSION 0.5.0
-ENV POETRY_VERSION 1.1.4
+ENV POETRY_VERSION 1.1.15
```

Comment:
Looks like we hit this problem: https://github.com/python-poetry/poetry/issues/6300

So bumping the poetry version.